### PR TITLE
comments added for godoc in cmd

### PR DESCRIPTION
**Pull request by shekhar2807**
# Description

-Comments added for cmd for the godoc documentation.
-To generate the documentation :

$ godoc -http=127.0.0.1:6060 

or just the port 

$ godoc -http=:6060

#687

### CODE REVIEW

File: cmd/addStake.go [hunk 1]
```diff
@@ -118,6 +123,7 @@ func (*UtilsStruct) StakeCoins(txnArgs types.TransactionOptions) (common.Hash, e
 	return transactionUtils.Hash(tx), nil
 }
 
+//This function add the following command to the root command
```
Comment (ashish10677) [inline cmd/addStake.go:126]: You can skip this description for all the commands.

**Review by ashish10677 — CHANGES_REQUESTED** [review]

File: cmd/addStake.go (hunk 1 above)
Comment (shekhar2807) [reply cmd/addStake.go]: Done @ashish10677

**Review by ashish10677 — APPROVED** [review]
LGTM

**Review by ashish10677 — APPROVED** [review]
LGTM

**Pull request merged**